.github/workflows/Release.yml

```diff
@@ -43,8 +45,13 @@ jobs:
   call-release-riseclipseweb-workflow:
     name: Release on riseclipseweb
     needs: [call-validate-release-workflow]
-    uses: riseclipse/riseclipse-developer/.github/workflows/Shared-Deploy-On-RiseClipseWeb.yml@master
+    uses: riseclipse/riseclipse-developer/.github/workflows/Shared-Release-On-RiseClipseWeb.yml@master
     with:
       artifact_path: riseclipse-validator-scl2003/fr.centralesupelec.edf.riseclipse.iec61850.scl.validator/target/RiseClipseValidatorSCL
```

Comment:
needs riseclipse/ in front of the path ?

Reply:
Changing it to looks more like Shared-Release-On-DockerHub.yml.